backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc

```diff
@@ -1,34 +1,46 @@
 
-:_mod-docs-content-type: PROCEDURE
+:_content-type: PROCEDURE
```

Comment:
Revert, also it seems like this type should be `ASSEMBLY` instead of `PROCEDURE`?

Reply:
@adellape 
I changed the attribute, but based on the other modules in the assembly this module is a procedure.

Reply:
I think what Alex is saying is that this is an assembly file based on the metadata and the fact that it's not in the modules section. Assembly files use the type of `ASSEMBLY`:

`The content type for the file. For assemblies, always use :_mod-docs-content-type: ASSEMBLY. Place this attribute before the anchor ID or, if present, the conditional that contains the anchor ID.`

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata